src/TLuaInterpreter.cpp

```diff
@@ -3042,202 +3042,53 @@ int TLuaInterpreter::setLabelClickCallback(lua_State* L)
             event.mArgumentTypeList.append(ARGUMENT_TYPE_NIL);
         } else {
             lua_pushfstring(L,
-                            "setLabelClickCallback: bad argument #%d type (boolean, number, string or nil\n"
+                            "%s: bad argument #%d type (boolean, number, string or nil\n"
                                     "expected, got a %s!)",
+                            funcName.toUtf8().constData(),
                             i,
                             luaL_typename(L, i));
             return lua_error(L);
         }
     }
 
-    if (mudlet::self()->setLabelClickCallback(&host, labelName, eventName, event)) {
+    bool lua_result;
+    if (funcName == QStringLiteral("setLabelClickCallback"))
```

Comment:
I think you can use `QLatin1String` rather than `QStringLiteral` in these - which sometimes has a microscopic bonus, I do not recall the exact details but I suspect @vadi2 can put it more precisely...!

Reply:
Let's not worry about it, this is good.